docs/UserGuide.md

```diff
@@ -69,7 +69,7 @@ Adds an item to the list of tracked inventory.
 Format: `addi n/ITEM_NAME q/QUANTITY d/DESCRIPTION [t/TAG]…​ sp/SELL_PRICE cp/COST_PRICE`
 
 <div markdown="span" class="alert alert-primary">:bulb: **Tip:**
-An inventory item can have any number of tags (including 0)
+An inventory item can have any number of tags (including 0). A tag should only consist of 1 word.
```

Comment:
Hmm perhaps can also add that for tags with more than one word, perhaps they can use pascal case instead. something like `If you would like to add tags with more than one word, use pascal-casing, e.g. DoThisInstead`